### ISSUE TITLE
Address bar contains `about:blank` after blank page is loaded

### ISSUE THREAD

**Issue by MrDOS** [bbo2adwuff/BlankPrivateBrowsingPage#4]
When I manually navigate to `about:blank`, after the page loads, the contents of the address bar are replaced with the default prompt text (“Search with _provider_ or enter address”). However, when this add-on causes a tab to navigate to the blank page, the `about:blank` URL stays in the address bar. In both cases, the address bar has focus and the caret is present at the end of the field. In the Blank Private Browsing Page case, this causes a convenience/usability issue: because the `about:blank` text isn't selected, I can't just start typing a URL (or search term, or...) and mash Enter: I need to first select the contents of the field so that it will be overwritten.

This is new as of the last few major versions of Firefox. Sorry, I don't remember when it started happening. Not until several versions post-Quantum, though.

**Comment by bbo2adwuff** [comment]
True, I realized it myself. At the moment I am not sure what caused the change.

But I'll have a look.
